src/client/main.tsx

```diff
@@ -5,17 +5,27 @@ import { GameState } from '../core/game';
 import { ClientController } from './client-controller';
 import { Splash } from './splash';
 
+const GAME_START_TIME = 5;
```

Comment:
From what I understand, the server will also need to know this value, as it also does a countdown using 5 seconds, so this variable should be put somewhere shared so that both can use it.